### DISCUSSION TITLE
1.0.4

### DISCUSSION THREAD

**Discussion by boris-jenicek** [boris-jenicek/ngx-awesome-popup#6] [Releases]
bugfix (thanks to @rmcsharry)
* preventing multiple events from emitting when the popup is closed
* added completed action to observables (not need for unsubscription anymore)

<hr /><em>This discussion was created from the release <a href='https://github.com/costlydeveloper/ngx-awesome-popup/releases/tag/1.0.4'>1.0.4</a>.</em>

**Comment by rmcsharry** [comment]
@costlydeveloper That's great that you fixed it so quickly. Great job!
